src/state/reducers/index.js

```diff
@@ -63,3 +63,27 @@ export const getLegacyChapterAlignments = (state, chapter) =>
  */
 export const getIsVerseAligned = (state, chapter, verse) =>
   fromAlignments.getIsVerseAligned(state.tool.alignments, chapter, verse);
+
+export function getManifest(state) {
+  return state.projectDetailsReducer.manifest;
+}
+
+export function getProjectSaveLocation(state) {
```

Comment:
not processing state. This should remain in the component.

Reply:
Done.